exercises/run-length-encoding/example.el

```diff
@@ -0,0 +1,62 @@
+;;; run-length-encoding.el --- run-length-encoding Exercise (exercism)
+
+;;; Commentary:
+
+;;; Code:
+
+
+(defun run-length-group (s)
+  "return list of character run counts"
+  (let ((case-fold-search nil)
+	groups)
+    (with-temp-buffer
```

Comment:
Could this be accomplished with a data structure in memory rather than a temp-buffer?

Reply:
Sure, that is always an option.  In fact, my first try did use an in-memory structure but it was more complex.  This version is much simpler, and it pushes the work of grouping letters down into the regular expression code so it should be faster.   Simpler and faster seams like a win to me.

Reply:
Alright, Sounds like a good enough argument to me 👍